### DISCUSSION TITLE
Raspberry Pi 5 Support

### DISCUSSION THREAD

**Discussion by danic85** [makerforgetech/modular-biped#56] [Ideas]
I have been using a Raspberry Pi 3b+ in the head on my build as it was the latest version when I started the project. I skipped the 4 just because by that point I had the Coral USB Accelerator to make the face recognition faster and that was the only computationally intense process at the time.

I'm interested in testing out the Pi 5 when it is released to see how it compares to the current hardware and it may remove the need for the USB accelerator, which would make this more accessible. I need to redesign the head for the latest components so it's a good opportunity to swap it out.

I'm interested to hear if anyone else is planning on using the Pi 5 in their build?

**Comment by Chompalot** [comment]
I'm concentrating on a fairly vanilla build at the moment. I have pre ordered the Pi 5 though.

**Comment by WhoIsMrSentry** [comment]
I will still continue with Jetson Nano, but if there is anything I can help with, I will be happy to help and if you want, I can make a 3D model of the head.

**Comment by WhoIsMrSentry** [comment]
I started drawing the head for the raspberry pi 5. Since there is no ready RPI5 CAD model, I made a drawing myself and I am leaving it attached in case you want to make a new head model.

[RPI_5_Simple.step](https://github.com/dmt-labs/modular-biped/files/12782277/RPI_5_Simple.zip)

**Reply by danic85** [reply]
Thanks for this! I have a few other things to try before the Pi arrives but this will definitely be helpful :clap:

**Comment by TheHammerMan00** [comment]
I have the pi5 on pre order now.  I thought about the 3b+ for the sake of flowing along but I'm thinking it wont be to long before most people use the pi5 anyway.

**Reply by danic85** [reply]
Yeah the plan is to keep the code for the 3b+ in a branch but continued development will be on the Pi 5, hopefully without the need for the coral usb accelerator (to make it more accessible).

I'm waiting for mine to arrive then I'll be doing an upgrade.
